core/orchestrator_core.go

```diff
@@ -283,6 +346,16 @@ func (o *TridentOrchestrator) bootstrapStorageClasses(ctx context.Context) error
 
 // Updates the o.volumes cache with the latest backend data. This function should only edit o.volumes in place to avoid
 // briefly losing track of volumes that do exist.
+// bootstrapVolumes adds existing volumes to the orchestrator
```

Comment:
Would the existing comments have been recognized as function documentation and formatted as such by godoc? In my code I assumed that if the comment block didn't start with "// <function name>" then it was just comments rather than actual doc. I can change that and assume the opposite (and expand with parameters, returns, example)